### PR TITLE
feat(ui5-slider, ui5-range-slider): SAP Horizon theme implementation

### PR DESCRIPTION
**Pull request by ndeshev**

### CODE REVIEW

**Review by niyap — CHANGES_REQUESTED** [review]
1.  The start point of ui5-slider component, background color is not correct.
2. While moving the whole progress bar of ui5-range-slider, the handles should have "active" styling, not as hovered.
3. Handles in compact mode look really small and ugly.
4. In compact mode the offset between the focus outline and the handles is too big.

File: packages/main/src/themes/SliderBase.css [hunk 1]
```diff
@@ -65,10 +66,19 @@
...
-	width: 0.5rem;
-	height: 0.5rem;
+	width: 0.375rem;
+	height: 0.375rem;
+	right: -0.875rem;
+	top: -0.0625rem;
+}
+
+[dir="rtl"] .ui5-slider-progress-container::after {
+	background-color: #1B90FF;
```
Comment (elenastoyanovaa) [inline packages/main/src/themes/SliderBase.css:76]: don't we have parameters for this?

File: packages/main/src/themes/SliderBase.css [hunk 2]
```diff
@@ -65,10 +66,19 @@
...
 	right: -0.875rem;
-	top: -0.125rem;
+}
+
+:host([ui5-range-slider]) .ui5-slider-progress-container::before {
+	background-color: #8396A8;
```
Comment (elenastoyanovaa) [inline packages/main/src/themes/SliderBase.css:81]: and this

File: packages/main/src/themes/SliderBase.css [hunk 3]
```diff
@@ -88,8 +99,20 @@
 	height: var(--_ui5_slider_outer_height);
 }
 
+:host([ui5-range-slider]) .ui5-slider-progress:focus::after {
+	border: 0.125rem solid #0070f2;
```
Comment (elenastoyanovaa) [inline packages/main/src/themes/SliderBase.css:103]: parameter?

File: packages/main/src/themes/sap_belize/SliderBase-parameters.css [hunk 4]
```diff
@@ -9,10 +9,10 @@
 	--_ui5_slider_handle_width: 1.75rem;
 	--_ui5_slider_handle_border: solid 0.125rem #999;
 	--_ui5_slider_handle_background: rgba(var(--sapField_Background), 0.1);
+	--_ui5_slider_handle_hover_background: rgba(217,217,217,0.6);
```
Comment (elenastoyanovaa) [inline packages/main/src/themes/sap_belize/SliderBase-parameters.css:12]: parameter?

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
I noticed that many of the values are without parameters and this is strange especially for colors. Do we also use hardcoded colors in UI5? If we used parameters, maybe some of the things would come out of the box and there would be no need to overwrite them in eachc theme. If no parameters are available, ignore my comment.

File: packages/main/src/themes/sap_belize/SliderBase-parameters.css (hunk 4 above)
Comment (ndeshev) [reply packages/main/src/themes/sap_belize/SliderBase-parameters.css:12]: In UI5 specifications is with LESS calculation fade(darken(@sapUiFieldBackground, 15), 60)

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**